packages/terra-form/src/Control.scss

```diff
@@ -0,0 +1,23 @@
+@import '~terra-mixins';
+
+.terra-Form-control {
+  @include terra-margin-end(0.714em);
+
+  display: block;
+  margin-bottom: 0.714em;
+  width: auto;
+
+  .terra-Form-input {
+    vertical-align: middle;
+  }
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
When doing the selection styles in table, Neil removed the logic for a responsive table in tiny mode. His reasoning was the base component should have a standard display in all viewports. Does this concept apply here? @neilpfeiffer